lib/ModuleParseError.js

```diff
@@ -13,11 +13,17 @@ class ModuleParseError extends WebpackError {
 	 * @param {Module} module the errored module
 	 * @param {string} source source code
 	 * @param {Error&any} err the parse error
+	 * @param {any} loaders the loaders used
```

Comment:
We can use `string[]` instead of `any`.
```suggestion
	 * @param {string|string[]} loaders the loaders used
```